### PR TITLE
completed css positioning exercise

### PR DESCRIPTION
**Pull request by KiraHike**
image
image
image

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 10px;
+  left: 220px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:17]: Please revise this to use smaller measurements from `right` as we discussed in Q&A

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 160px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:29]: Same here, use `bottom`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far! Please see my notes:

**Comment by KiraHike** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**